src/DFG.cpp

```diff
@@ -219,8 +219,13 @@ void DFG::combineForIter(list<string>* t_targetPattern){
 }
 
 void DFG::combineForUnroll(list<string>* t_targetPattern){
-  // combineForUnroll is used to reconstruct "phi-add-add-..." alike patterns provided by users which should be a cycle, otherwise, the fusion won't be performed.
+  // combineForUnroll is used to reconstruct "phi-add-add-..." alike patterns, otherwise, the fusion won't be performed.
   int patternSize = t_targetPattern->size();
+  if (patternSize > 4){ 
+    std::cout<<"\033[32m [Erro] The pattern is too long! \033[0m" <<std::endl;
```

Comment:
Typo: Erro -> Error.
You can change it to "[ERROR] we currently only support pattern with length less than 5."